releases/v1.0.2-rc.0.toml

```diff
@@ -0,0 +1,27 @@
+# commit to be tagged for new release
+commit = "HEAD"
+
+project_name = "containerd"
+github_repo = "containerd/containerd"
+
+# previous release
+previous = "v1.0.1"
+
+pre_release = true
+
+preface = """\
+This is the second patch release for `containerd` in the 1.0 series. There are
+several fixes to reduce memory usage and garbage collection pressure in the
+shim. The content store has a few fixes to address cases where download
+resumption wasn't working properly. The image store has fixes to be more
+efficient and to handle edge cases in the garbage collector.
+
+The most critical fix addresses a hang in runc that can happen due to a race
+condition around FIFOs. Make sure to update your runc binary if you are
```

Comment:
Maybe link to the revision or PR from/where which it was fixed since `runc` still doesn't have releases yet.

Reply:
@crosbymichael any details here?

Reply:
https://github.com/opencontainers/runc/pull/1698